23&22/spider.py

```diff
@@ -34,9 +34,8 @@
 
     span = aTag.find("span")
     print(span.text)
-    #span.replace("'", " ")
     print()
-    cur.execute('INSERT INTO foods (name, url) VALUES (?, ?)', (span.text.replace("'", " "),foodUrl))
+    cur.execute('INSERT INTO foods (name, url) VALUES (?, ?)', (span.text.replace(" ", "_").replace('U+00a0', ""),foodUrl))
```

Comment:
If you replace space with _, you would change the original name. Why do you need to replace? Let's look at what you really want to do here.